LUA : delay esc prearm check on safety disarm

Pradeep-Carbonix opened the pull request:
This delay is introduced to prevent one-time ESC telemetry lost and recovered message, when the aircraft safety is disarmed

robertlong13 commented on libraries/AP_HAL_ChibiOS/hwdef/CarbonixCommon/scripts/modules/bit_esc.lua:
```diff
@@ -90,6 +95,12 @@ function ESC:update()
+        return
+    end
+
+    -- Wait for a minute before checking for errors
```
Tidy up this comment, but otherwise this is all good.

Pradeep-Carbonix replied:
Updated.

Pradeep-Carbonix committed:
LUA : delay esc prearm check on safety disarm

This delay is introduced to prevent one-time ESC telemetry lost and recovered message, when the aircraft safety is disarmed
